### ISSUE TITLE
Notifications with certain infos about tracking

### ISSUE THREAD

**Issue by Tagirijus** [almarklein/timetagger#460]
I used a time tracker on my mobile, which shows which tasks runs and how long was tracked already. This gets quite handy, because do not need to unlock my mobile (only let me show the lock screen) to see how long I am working already. Would something like this would technically be possible?

Other notification options would be nice as well. Also if the user could choose which to see there. I guess some ideas are already in the issues here, right? I just could not find a dedicated notification-feature-issue. Notification ideas in general:

- Description and tracked duration so far
- Start time
- Last tracked tasks

Maybe other users have ideas as well.

**Comment by Tagirijus** [comment]
I mean the notifications and not a home screen widget. The later one would not be possible at all anyway, right? Yet I am only referring to a notification you get on the lock-screen or when you drag down this thing from above (how is it officially called? hope you know what I mean ... the notification panel or so).

**Comment by almarklein** [comment]
> The later one would not be possible at all anyway, right?

No. Notifications are merely messages that an app can send to the OS. They cannot be altered afterwards. I *think* notifications can be closed though. So could emit a notification when a timer has started, and close it when the timer stops.

**Comment by Tagirijus** [comment]
Ah, I see. Well, yeah: fair enough to get at least one notification after all. Your idea sounds already great. Maybe it could contain the start time at least and the user can do the math on its own when looking at the notification then.

Thanks for considering - despite if it will be implemented or not some day. (;
